addons/website/static/src/snippets/s_popup/001.scss

```diff
@@ -10,13 +10,13 @@
     .s_popup_close {
         z-index: $zindex-modal;
         @include o-position-absolute(0, 0);
-        width: $font-size-lg * 2;
-        height: $font-size-lg * 2;
-        line-height: $font-size-lg * 2;
+        width: $h1-font-size;
```

Comment:
> We can keep font-size-lg in our code if you prefer, just know that the user has no control over it

Yes, it would be better. No worries for the lack of control on `font-size-lg`; as long as it remains proportional to `font-size-base` we're good 👍 🙂

Reply:
Ok let's remove that part of the spec then @Guillaume-gdi, I did not gave it enough thoughts originally.

Reply:
I revert it :+1: